Nitrox.Launcher/Nitrox.Launcher.csproj

```diff
@@ -1,7 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>$(NitroxNetCoreVersion)</TargetFramework>
```

Comment:
`TargetFramework` is very project-specific and only used in Nitrox.Launcher at the moment. Would prefer no additional code unless there's at least 2 or more instances where it is used.

Reply:
I intend to port the server and buildtool to net6 as well. Would this variable still be worth keeping around?

Reply:
Variable removed in newest commit. In most cases however, you want to avoid multiple versions of .NET Core, as it causes more burden to the users who need to have multiple runtimes installed.